lib/auth/saml_test.go

```diff
@@ -87,3 +93,98 @@ func (s *SAMLSuite) TestCreateSAMLUser(c *check.C) {
+
+// This XML blob is a sample EntityDescriptor made to satisfy the connector validator for testing.
+const EntityDescriptor = `<EntityDescriptor ID="_2240bd9c-30c4-4d2a-ab3e-87a94ea334fd" entityID="http://some.entity.id"
```

Comment:
nit: move these inside of the test body to not pollute the package-level namespace

Reply:
adjusted